src/document/DocumentAnnotator.ts

```diff
@@ -102,6 +102,12 @@ export default class DocumentAnnotator extends BaseAnnotator {
         } else {
             this.annotatedEl.classList.remove('ba-is-highlighting');
         }
+
+        if (mode === Mode.REGION) {
+            this.annotatedEl.classList.add('ba-is-create-region');
+        } else {
+            this.annotatedEl.classList.remove('ba-is-create-region');
+        }
```

Comment:
Maybe this could be more scalable with drawing mode also on the horizon, something like:
```
<remove all classes from annotatedEl>
switch (mode) {
    case Mode.HIGHLIGHT:
        this.annotatedEl.classList.add('ba-is-highlighting');
        break;
...
}
```